java/core/src/main/java/sleeper/core/properties/table/TablePropertiesStore.java

```diff
@@ -72,8 +74,9 @@ public TableProperties loadByName(String tableName) {
     /**
      * Loads and validates properties for the Sleeper table with the given unique ID.
      *
-     * @param  tableId the unique table ID
-     * @return         the table properties
+     * @param  tableId                the unique table ID
+     * @return                        the table properties
+     * @throws TableNotFoundException if the table for the given ID is not found.
```

Comment:
This description doesn't need a full stop, as it's not a sentence.

Should this be "with the given ID" rather than "for the given ID"?